compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs

```diff
@@ -116,13 +120,20 @@ featureNatSynonyms = Feature
     , featureCppFlag = Just "DAML_NAT_SYN"
     }
 
-featureInterfaces :: Feature
-featureInterfaces = Feature
-    { featureName = "Daml Interfaces"
-    , featureMinVersion = versionDev
+featureSimpleInterfaces :: Feature
+featureSimpleInterfaces = Feature
+    { featureName = "Daml Interfaces without guards and requires"
```

Comment:
no strong feeling here but I think it's a bit weird to define a feature in terms of what it doesn't have

```suggestion
    { featureName = "Daml Interfaces"
```